### PR TITLE
Add medium type text

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
@calvinjia PTAL

File: docs/en/advanced/Alluxio-Storage-Management.md [hunk 1]
```diff
@@ -100,6 +100,16 @@ alluxio.worker.tieredstore.level0.dirs.quota=16GB,100GB,100GB
 
 Note that the ordering of the quotas must match with the ordering of the paths.
 
+We also need to specify what kind of storage medium each path is on. Our previous example had one path that is ramdisk, followed by two SSDs. We can indicate that by using:
+
+```
+alluxio.worker.tieredstore.level0.dirs.mediumtype=MEM,SSD,SSD
```
Comment (calvinjia) [inline docs/en/advanced/Alluxio-Storage-Management.md]: Should this be updated in the other code blocks?

Comment (yuzhu) [reply docs/en/advanced/Alluxio-Storage-Management.md]: moved to the other code block. 
PTAL

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**